clippy.toml

```
@@ -0,0 +1,3 @@
+disallowed-types = [
+  { path = "axum::extract::Path", reason = "use our own custom web::extractors::Path for a nicer error response" },
+]
```

```suggestion
[[disallowed-types]]
path = "axum::extract::Path"
reason = "use our own custom web::extractors::Path for a nicer error response"
```

non-inline tables are much easier to read with long strings